uphold-integration/src/main/java/org/dash/wallet/integration/uphold/data/UpholdConstants.java

```diff
@@ -2,12 +2,12 @@
 
 public class UpholdConstants {
 
-    public static final String CLIENT_ID = "dfb85d44118d6ca2b3e070d434da6e9102a3c7d9";
-    public static final String CLIENT_SECRET = "fdb513ff0dd2672a23875816d31354076fc5372e";
-    public static final String CLIENT_BASE_URL = "https://api-sandbox.uphold.com/";
+    public static final String CLIENT_BASE_URL = "https://api.uphold.com/";
+    public static final String CLIENT_ID = "c184650d0cb44e73d8e5cb2021753a721c41f74a";
+    public static final String CLIENT_SECRET = "da72feee8236f7709df6d0c235a8896ad45f2a91";
 
-    public static final String INITIAL_URL = "https://sandbox.uphold.com/authorize/dfb85d44118d6ca2b3e070d434da6e9102a3c7d9?scope=accounts:read%%20cards:read%%20cards:write%%20transactions:deposit%%20transactions:read%%20transactions:transfer:application%%20transactions:transfer:others%%20transactions:transfer:self%%20transactions:withdraw%%20transactions:commit:otp%%20user:read&state=%s";
-    public static final String CARD_URL_BASE = "https://sandbox.uphold.com/dashboard/cards/%s/add";
-    public static final String TRANSACTION_URL = "https://sandbox.uphold.com/reserve/transactions/%s";
+    public static final String INITIAL_URL = "https://uphold.com/authorize/c184650d0cb44e73d8e5cb2021753a721c41f74a?scope=accounts:read%%20cards:read%%20cards:write%%20transactions:deposit%%20transactions:read%%20transactions:transfer:application%%20transactions:transfer:others%%20transactions:transfer:self%%20transactions:withdraw%%20transactions:commit:otp%%20user:read&state=%s";
```

Comment:
Is it possible to use `CLIENT_ID` here instead of hard-coding this value again?

Reply:
Yes, it can me.  However, this code shouldn't be part of this PR.  It is not related to logging of http calls.